### PR TITLE
Order topics by creation instead of update

### PR DESCRIPTION
**Pull request by notriddle**
This prevents topics from being un-published

https://discourse.janitor.technology/t/our-latest-newsletter-janitor-news-11-keeps-getting-unpublished/241

### CODE REVIEW

File: lib/blog.js [hunk 1]
```diff
@@ -14,8 +14,7 @@ const metrics = require('./metrics');
 // [^1] "published" is a staff-only-tag.
 // [^2] "blog" is a category, and is not staff-only.
 
-const discourseBlogCategoryID = 9;
-const discourseBlogTopicsUrl = 'https://discourse.janitor.technology/tags/published';
+const discourseBlogTopicsUrl = 'https://discourse.janitor.technology/tags/published?category=9&parent_category=8&order=created&ascending=false';
```
Comment (jankeromnes) [inline lib/blog.js]: Nit: When visiting https://discourse.janitor.technology/tags/published?category=9&parent_category=8&order=created&ascending=false, the post order looks mostly right, except:

- `Janitor Alpha is here!` seems to be the second most recent
- `Janitor News #9` is all the way down as the oldest

Comment (jankeromnes) [reply lib/blog.js]: Also, nit, shouldn't that be `published.json`, since you've removed `.json` below?

Comment (notriddle) [reply lib/blog.js]: Looks like it was a problem with the old article migration. It's correctly sorted now.

Comment (jankeromnes) [reply lib/blog.js]: Newsletter 9 is indeed back in the right place, but not Janitor Alpha. But that can probably be fixed in Discourse, and is orthogonal to this pull request.

**Pull request merged**